apps/src/lab2/views/Loading.tsx

```diff
@@ -23,10 +24,22 @@ const Loading: React.FunctionComponent<LoadingProps> = ({
     ? moduleStyles.fadeLoading
     : moduleStyles.fadeLoaded;
 
+  const background = useAppSelector(
+    state =>
+      state.progress.lessons?.find(l => l.id === state.progress.currentLessonId)
+        ?.background || null
+  );
+
   return (
     <div
       id="fade-overlay"
-      className={classNames(moduleStyles.solidBlock, overlayStyle)}
+      className={classNames(
+        moduleStyles.solidBlock,
+        background === 'light'
+          ? moduleStyles.solidBlockLight
+          : moduleStyles.solidBlockDark,
```

Comment:
super nit: for simplicity, you could also rename the classes to `solid-block-light/dark` and dos something like ``moduleStyles[`solid-block-${background}`]``

Reply:
Good call.